pkg/kubelet/cm/container_manager.go

```diff
@@ -212,6 +224,30 @@ func int64Slice(in []int) []int64 {
 	return out
 }
 
+func podHasExclusiveCPUs(cr cpuAllocationReader, pod *v1.Pod) bool {
+	for _, container := range pod.Spec.InitContainers {
+		if containerHasExclusiveCPUs(cr, pod, &container) {
+			return true
+		}
+	}
+	for _, container := range pod.Spec.Containers {
+		if containerHasExclusiveCPUs(cr, pod, &container) {
+			return true
+		}
+	}
+	klog.V(4).InfoS("Pod contains no container with pinned cpus", "podName", pod.Name)
+	return false
+}
+
+func containerHasExclusiveCPUs(cr cpuAllocationReader, pod *v1.Pod, container *v1.Container) bool {
+	exclusiveCPUs := cr.GetExclusiveCPUs(string(pod.UID), container.Name)
+	if !exclusiveCPUs.IsEmpty() {
+		klog.V(2).InfoS("Container has pinned cpus", "podName", pod.Name, "containerName", container.Name)
```

Comment:
for the reasons outlined above I think this should be reverted to V=4; we added the required logs with V=2, so we should be good.

Reply:
Still convinced because we can possibly emit a log per container with pinned CPUs, which is bordering "too much" (cc @SergeyKanzhelev )

Reply:
reverted to v4. still helpful to have option for debugging this during featuregate period no?